src/NzbDrone.Core/Indexers/Fanzub/FanzubSettings.cs

```diff
@@ -25,6 +25,9 @@ public FanzubSettings()
         [FieldDefinition(0, Label = "Rss URL", HelpText = "Enter to URL to an Fanzub compatible RSS feed")]
         public string BaseUrl { get; set; }
 
+        [FieldDefinition(1, Label = "Anime seasonal search", Type = FieldType.Checkbox, HelpText = "Add seasonal format search for Anime")]
```

Comment:
This format is referred to as `Standard` everywhere else in Sonarr, it should be referred to that way in the setting as well, Season/Seasonal is far too easily confused with season pack searches.

Maybe: `Anime Standard Format Search` with the help text of `Also search for anime using the standard numbering format`